get-started/index.md

```diff
@@ -59,10 +62,320 @@ Click "Install" and wait for the installation to complete.
+the Cryostat Agent:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+...
+spec:
+  ...
+  template:
+    ...
+    spec:
+      containers:
+        - name: sample-app
+          image: docker.io/myorg/myapp:latest
+          env:
+            - name: CRYOSTAT_AGENT_APP_NAME
+              value: "myapp"
+              # Replace this with the Kubernetes DNS record
+              # for the Cryostat Service
+            - name: CRYOSTAT_AGENT_BASEURI
+              value: "http://cryostat.mynamespace.mycluster.svc:8181"
+            - name: POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.PodIP
+            - name: CRYOSTAT_AGENT_CALLBACK
+              value: "http://$(POD_IP):9977"
+              # Replace "abcd1234" with a base64-encoded authentication token
+            - name: CRYOSTAT_AGENT_AUTHORIZATION
+              value: "Bearer abcd1234"
+          ports:
+            - containerPort: 9977
+              protocol: TCP
+          resources: {}
+      restartPolicy: Always
+status: {}
+```
+
+Port number `9977` is the default HTTP port that the Agent exposes for its internal webserver that services Cryostat requests. The `CRYOSTAT_AGENT_AUTHORIZATION` value is particularly
+noteworthy: these are the credentials that the Agent will include in API requests it makes to Cryostat to advertise its own presence. You should create a Kubernetes `Service Account` for
+this purpose and replace `abcd1234` with the base64-encoded authentication token associated with the service account. For testing purposes you may use your own user account's
+authentication token, for example with `oc whoami --show-token`.
+
+Finally, create a `Service` to enable Cryostat to make requests to this Agent:
+
+```yaml
+apiVersion: v1
+kind: Service
+...
+spec:
+  ports:
+    - name: "cryostat-agent"
+      port: 9977
+      targetPort: 9977
+...
+```
+
+More details about the configuration options for the Cryostat Agent [are available here](https://github.com/cryostatio/cryostat-agent/blob/main/README.md#configuration).
+
+#### [Using JMX](#using-jmx)
+Cryostat is also able to use Java Management Extensions (JMX) to communicate with target applications. This is a standard JDK feature that can be enabled by passing JVM
+flags to your application at startup. A basic and insecure setup suitable for testing requires only the following three flags:
+
+```
+-Dcom.sun.management.jmxremote.port=9091
+-Dcom.sun.management.jmxremote.ssl=false
+-Dcom.sun.management.jmxremote.authenticate=false
+```
+
+[comment]: # TODO explain how to configure SSL and auth for JMX, or link to external docs
+
+In a real scenario you should enable both SSL and authentication on your application. You can then [trust the certificate](/guides/#add-a-trusted-certificate)
```

Comment:
rather than "real scenario", suggest using "It is recommended that you enable both SSL and authentication on your application."